app/data/sqlite3/db.go

```diff
@@ -6,19 +8,23 @@ import (
 
 	"github.com/jmoiron/sqlx"
 	// load sqlite3 library with side effects
-	_ "github.com/mattn/go-sqlite3"
+	sq3 "github.com/mattn/go-sqlite3"
 )
 
+type Error = sq3.Error
```

Comment:
Pushing these aliases here just means we can isolate the CGO requiring files and wont' need another  `_nocgo.go` file elsewhere